numpy/lib/tests/test_function_base.py

```diff
@@ -1641,6 +1641,23 @@ def test_simple(self):
         # check that unwrap maintains continuity
         assert_(np.all(diff(unwrap(rand(10) * 100)) < np.pi))
 
+    def test_minmax(self):
+        # check that unwrap removes jumps greater that 255
+        assert_array_equal(unwrap([1, 1 + 256], min_val=0, max_val=255), [1, 2])
+        # check that unwrap maintains continuity
+        assert_(np.all(diff(unwrap(rand(10) * 1000, min_val=0, max_val=255)) < 255))
+        # check simple case
+        simple_seq = np.linspace(0, 512, 6)
+        wrap_seq = np.mod(simple_seq, 255)
+        assert_array_equal(unwrap(wrap_seq, min_val=0, max_val=255), simple_seq)
+        # check custom discont value
+        uneven_seq = np.array(simple_seq.tolist()+[765])
+        wrap_uneven = np.mod(uneven_seq, 255)
+        no_discont = unwrap(wrap_uneven, min_val=0, max_val=255)
+        assert_array_equal(no_discont[:-1], uneven_seq[:-1])
+        sm_discont = unwrap(wrap_uneven, min_val=0, max_val=255, discont=2)
+        assert_array_equal(sm_discont, uneven_seq)
```

Comment:
I'd like to see a test where the expected value is written out explicitly - I'm struggling to understand the behavior of `discont` from this test.

Reply:
Ok I'll change the tests and sequences

Reply:
Thanks!